apps/node-red/docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: "3.7"
+
+services:
+  web:
+    image: nodered/node-red:2.0.6-12
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      # Replace <port> with the port that your app's web server
+      # is listening inside the Docker container. If you need to
+      # expose more ports, add them below.
+      - 1880:1880
+    volumes:
+      # Uncomment to mount your data directories inside
+      # the Docker container for storing persistent data
+      - ${APP_DATA_DIR}/data:/data
+      # - ${APP_DATA_DIR}/bar:/bar
+
+      # Uncomment to mount LND's data directory as read-only
+      # inside the Docker container at path /lnd
+      # - ${LND_DATA_DIR}:/lnd:ro
+
+      # Uncomment to mount Bitcoin Core's data directory as
+      # read-only inside the Docker container at path /bitcoin
+      # - ${BITCOIN_DATA_DIR}:/bitcoin:ro
+    environment:
+      # Pass any environment variables to your app for configuration in the form:
+      # VARIABLE_NAME: value
+      #
+      # Here are all the Umbrel provided variables that you can pass through to
+      # your app to connect to Bitcoin Core, LND, Electrum and Tor:
+      #
+      # Bitcoin Core environment variables
+      BITCOIN_NETWORK: $BITCOIN_NETWORK
+      BITCOIN_IP: $BITCOIN_IP
+      # $BITCOIN_P2P_PORT - P2P port
+      BITCOIN_RPC_PORT: $BITCOIN_RPC_PORT
+      BITCOIN_RPC_USER: $BITCOIN_RPC_USER
+      BITCOIN_RPC_PASS: $BITCOIN_RPC_PASS
+      # $BITCOIN_RPC_AUTH - RPC auth string
+      #
+      # LND environment variables
+      LND_IP: $LND_IP
+      LND_GRPC_PORT: $LND_GRPC_PORT
+      # $LND_REST_PORT - REST Port of LND
+      #
+      # Electrum server environment variables
+      ELECTRUM_IP: $ELECTRUM_IP
+      ELECTRUM_PORT: $ELECTRUM_PORT
```

Comment:
Why are you exposing those? Is there an easy way to connect from node-red to LND? If there is, you'd also need LND's data dir, otherwise you can remove these variables

Reply:
@AaronDewes 
I don't think anyone has implemented an LND node yet, but I want to implement one similar to https://flows.nodered.org/node/node-red-contrib-lightning (which connects to c-lightning). I will add a read-only access to the LND data dir, I didn't know it was necessary. In what scenario do you need to read files from the LND data dir, and the grpc commands aren't enough? I don't have much experience with that grpc api.

My end goal is to use something like https://flows.nodered.org/node/node-red-node-twitter and build a bot that listens to twitter hashtags and replies on twitter with a lightning invoice. Could be really cool and useful.

Reply:
Added the /bitcoin dir too.